daprdocs/content/en/reference/components-reference/supported-bindings/zeebe-command.md

```diff
@@ -0,0 +1,583 @@
+﻿---
+type: docs
+title: "Zeebe command binding spec"
+linkTitle: "Zeebe command"
+description: "Detailed documentation on the Zeebe command binding component"
+aliases:
+- "/operations/components/setup-bindings/supported-bindings/zeebe-commands/"
```

Comment:
```suggestion
```

Note needed as this is a net-new page and not migrated from an existing location